examples/experimental/CrypTen/CrypTen - Traing Encrypted Neural Network - Plans.ipynb

```diff
@@ -0,0 +1,543 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## CrypTen - Training an Encrypted Neural Network across Workers using Plans\n",
+    "\n",
+    "\n",
+    "We will train an encrypted neural network across different PySyft workers (deployed as [GridNodes](https://github.com/OpenMined/PyGridNode)) where each worker has a subset of the features we need for the train. For this we will be using Plans and we will be using CrypTen as a backend for SMPC. \n",
+    "\n",
+    "\n",
+    "Authors:\n",
+    " - George Muraru - Twitter: [@gmuraru](https://twitter.com/georgemuraru)\n",
+    " - Ayoub Benaissa - Twitter: [@y0uben11](https://twitter.com/y0uben11)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Training Overivew\n",
+    "* In this tutorial we will use the [MNIST dataset](http://yann.lecun.com/exdb/mnist/)\n",
+    "* The features we need for training the network are split accross two workers (we will name them *alice* and *bob*)\n",
+    "\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "\n",
+    "### Download/install needed repos\n",
+    "* Install PySyft from the [`crypten` branch](https://github.com/OpenMined/PySyft/tree/crypten).\n",
+    "* Clone the [GridNode repository](https://github.com/OpenMined/GridNode)\n",
+    "  * we need this because *alice* and *bob* are two different GridNodes\n",
+    "\n",
+    "### Run the grid nodes\n",
+    "* In two separate terminals run:\n",
+    "    * ```python -m gridnode --id alice --port 3000```\n",
+    "    * ```python -m gridnode --id bob --port 30001```\n",
+    "    \n",
+    "This will start two workers, *alice* and *bob* and we will connect to them using the port 3000 and 30001.\n",
```

Comment:
```suggestion
    "    * ```python -m gridnode --id bob --port 3001```\n",
    "    \n",
    "This will start two workers, *alice* and *bob* and we will connect to them using the port 3000 and 3001.\n",
```